frontend/src/app/app-routing.module.ts

```diff
@@ -1,7 +1,11 @@
 import { NgModule } from '@angular/core';
 import { RouterModule, Routes } from '@angular/router';
+import { ContentComponent } from './modules/workspace/content/content.component';
 
-const routes: Routes = [];
+const routes: Routes = [
+  { path: '', redirectTo: '', pathMatch: 'full' },
+  {path:'content', component:ContentComponent}
```

Comment:
Also. instead of loading the component, you should lazy load a workspace module: https://angular.io/guide/lazy-loading-ngmodules#lazy-loading-basics. Take some to read that page.
You'll also need a separate routing file for that module where you gonna specify routes